daemon/runtime_unix.go

```diff
@@ -63,6 +66,18 @@ func defaultV1ShimConfig(conf *config.Config, runtimePath string) *types.ShimCon
 	}
 }
 
+func kataV2ShimConfig(conf *config.Config, runtimePath string) *types.ShimConfig {
+	return &types.ShimConfig{
+		Binary: linuxKataShimV2,
+		Opts: &v2runcoptions.Options{
```

Comment:
Does kata really support this struct?

Reply:
thanks @AkihiroSuda -, I think that kata doesn't support the opts part. I have removed these lines.